jupyterlab/commands.py

```diff
@@ -356,7 +364,7 @@ def install_extension(self, extension, existing=None):
                 uninstalled.remove(extension)
                 config['uninstalled_core_extensions'] = uninstalled
                 self._write_build_config(config)
-            return
+            return False
```

Comment:
The return value is used as a boolean in order to determine whether a rebuild is needed. This change makes that value explicit, and I'm thinking it should probably be `True` instead of `False`.